resources/public/api/conf/1.0/testdata/fund-release.md

```diff
@@ -0,0 +1,446 @@
+<table>
+    <col width="20%">
+    <col width="40%">
+    <col width="40%">
+    <thead>
+        <tr>
+            <th>Scenario</th>
+            <th>Request Payload</th>
+            <th>Response</th>
+        </tr>
+        <tr>
+              <td>
+                  <p>Fund release created</p>
+                  <p class="code--block">
+                  <strong>lisaManagerReferenceNumber:</strong><br>
+                  <a href="https://test-developer.service.hmrc.gov.uk/api-documentation/docs/api/service/lisa-api/1.0#testing-the-api">Use your test user profile</a><br>
+                   <br>
+                        <strong>accountId:</strong><br>1234567890
+                  </p>
+               </td>
+         <td>
+             <pre class="code--block">
+ {
```

Comment:
Can you move all the json examples right up to the left margin and try and reduce each indentation to two spaces?